client/src/util/stateManager/annotationsHelpers.js

```diff
@@ -132,6 +132,22 @@ export function setLabelByMask(df, colName, mask, label) {
 	return ndf;
 }
 
+export function allHaveLabelByMask(df, colName, label, mask) {
+	// return true if all rows as indicated by mask have the colname set to label.
+	// False if not.
+	const col = df.col(colName);
+	if (!col) return false;
+	if (df.length != mask.length)
```

Comment:
Double equals https://stackoverflow.com/questions/1946063/in-javascript-is-same-as, single equals considered unsafe

Reply:
fixed